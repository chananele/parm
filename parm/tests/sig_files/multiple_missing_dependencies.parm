---
name: missing_deps
imports:
  - a
  - b
exports:
  - c
pattern: |
  0x1000: mov r0, r1
...
