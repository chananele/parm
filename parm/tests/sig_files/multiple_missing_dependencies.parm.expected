---
name: missing_deps
result: not run
errors:
  - unresolved import [a]
  - unresolved import [b]
...


